app/src/main/java/com/codingblocks/cbonlineapp/library/LibraryActivity.kt

```diff
@@ -19,6 +19,9 @@ class LibraryActivity : BaseCBActivity() {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_library)
         setToolbar(libraryToolbar)
+        libraryToolbar.setNavigationOnClickListener(){
```

Comment:
Nit change
`setNavigationOnClickListener{`

Reply:
means? Can you elaborate?

Reply:
oh I got it. 👍